scss/utils/helpers/_type.scss

```diff
@@ -73,6 +73,11 @@
 	}
 }
 
+// Normal text 
```

Comment:
thanks to the way `extend` works with the cascade, `%text--normal` is better off declared after the other text placeholders.